src/resources/js/components/enso/vuedatatable/VueTable.vue

```diff
@@ -355,6 +355,7 @@ export default {
                 intervals: this.intervals,
                 params: this.params,
                 selected: this.selected,
+                comparisonOperator: this.template.comparisonOperator,
```

Comment:
this need to be added to the `exportRequest()` method also. We want in the excel export the same data as we get in the table.